oort/oort-aws/src/main/groovy/com/netflix/spinnaker/oort/aws/provider/agent/ClusterCachingAgent.groovy

```diff
@@ -398,7 +398,20 @@ class ClusterCachingAgent implements CachingAgent, OnDemandAgent, AccountAware,
     def usableOnDemandCacheDatas = []
     providerCache.getAll(ON_DEMAND.ns, asgs.collect { Keys.getServerGroupKey(it.autoScalingGroupName, account.name, region) }).each {
       if (it.attributes.cacheTime < start) {
-        evictableOnDemandCacheDatas << it
+        if (account.eddaEnabled) {
```

Comment:
I contemplated that ... and it may still be worthwhile but from looking at clouddriver logs, we don't evict multiple server groups in a single caching cycle very often.